packages/ckan/test/ckan.test.js

```diff
@@ -0,0 +1,29 @@
+import assert from 'assert'
+import withServer from 'express-as-promise/withServer.js'
+import { describe, it } from 'mocha'
+import trifidFactory from '../src/index.js'
+
+const createTrifidConfig = (config, server = {}) => {
+  const loggerSpy = []
+
+  return {
+    logger: (str) => loggerSpy.push(str),
+    server,
+    config,
+  }
+}
+
+describe('@zazuko/trifid-plugin-ckan', () => {
+  describe('trifid factory', () => {
+    it('should be a factory', () => {
+      assert.strictEqual(typeof trifidFactory, 'function')
+    })
```

Comment:
TBH, I never understood these tests. ou call the function in all other cases. Why explicitly assert that it's a function?

Reply:
Yes agree, I will remove those tests.

Reply:
Done.